project_task_assignment/models/project.py

```diff
@@ -0,0 +1,47 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    employee_ids = fields.Many2many(
+        comodel_name='hr.employee',
+        relation='project_employee_rel',
+        column1='project_id',
+        column2='employee_id',
```

Comment:
Thanks for these modules.
Isn't it easier and concise code if you don't specify `relation, column1, column2` attributes